ansible/roles/web/templates/parameters.yml.j2

```diff
@@ -58,3 +58,7 @@ parameters:
 
     app.librecores_domain: '{{ librecores_domain }}'
     app.librecores_url: '{{ 'https' if use_https else 'http' }}://www.{{ librecores_domain }}'
+
+    # Notification mailer
+    notification_mailer_address: '{{ notification_mailer_address }}'
+    notification_sender_name: '{{ notification_sender_name }}'
```

Comment:
Please use consistent names for these two variables, and prefix 'app.'.

For example:

app.notification_from_address
app.notification_from_name